src/System.IO.Pipelines/src/System/IO/Pipelines/PipeCompletion.cs

```diff
@@ -68,12 +69,12 @@ public PipeCompletionCallbacks AddCallback(Action<Exception, object> callback, o
         [MethodImpl(MethodImplOptions.AggressiveInlining)]
         public bool IsCompletedOrThrow()
         {
-            if (_exception == null)
+            if (!_isCompleted)
             {
                 return false;
             }
 
-            if (_exception != s_completedNoException)
```

Comment:
Remove `s_completedNoException` now that it is no longer used.
https://github.com/dotnet/corefx/pull/31375/files#diff-ff9ac9c14f9484dcfcaa2332b6353541L16

Reply:
Done